russian/buildings-dep/tv/tv.md

```diff
@@ -1,4 +1,8 @@
 Вы сразу же попадаете на Первый канал, где идёт воскресная "Пока все дома". На столе в доме
 известной актрисы множество сладостей к чаю. Ваш взгляд невольно привлекает ваза с зефиром.
 Вы пристально всматриваетесь в белые объёмные фигуры и замечаете как одна зефиренка спрыгивает
-из вазы. Никто из гостей ничего не подозревает.
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.